test/unit/index.test.js

```diff
@@ -36,6 +37,16 @@ Test('logger', function (loggerTest) {
     assert.end()
   })
 
+  loggerTest.test('log error level, when filtered out', function (assert) {
+    const env = process.env
+    process.env.LOG_FILTER = 'info, debug'
+    const LoggerProxy = Proxyquire('../../src/index', {})
```

Comment:
Perhaps you should rather override the level in the proxy acquired as follows:
```
    const LoggerProxy = Proxyquire('../../src/index', {
customLevels: 'info, debug'
})
```

Then you don't need to directly override `process.env.LOG_FILTER`.

Let me know what you think.

Reply:
It doesn't work this way. Proxyquire allows you to override required modules, not variables/constants.